libs/coin-modules/coin-bitcoin/src/getAccountNetworkInfo.ts

```diff
@@ -4,7 +4,18 @@ import type { NetworkInfo } from "./types";
 import { getWalletAccount } from "./wallet-btc";
 import { Account } from "@ledgerhq/types-live";
 import { BitcoinInfrastructureError } from "./errors";
+import { getRelayFeeFloorSatVb } from "./wallet-btc/utils";
 const speeds = ["fast", "medium", "slow"];
+
+// Clamp each level to ≥ floor + ε and return integers
+export function clampFeesToFloor(
+  fees: BigNumber[],
+  floor: BigNumber,
+  epsilon: BigNumber = new BigNumber(1), // +1 sat/vB safety
+): BigNumber[] {
+  return fees.map(f => BigNumber.max(f, floor.plus(epsilon)).integerValue(BigNumber.ROUND_CEIL));
+}
```

Comment:
Rather than making the function doing the map, what about taking just a single value instead ? The caller can handle the map.
```suggestion
export function clamp(
  floor: BigNumber,
  epsilon: BigNumber = new BigNumber(1), // +1 sat/vB safety
): BigNumber => BigNumber {
  f => BigNumber.max(f, floor.plus(epsilon)).integerValue(BigNumber.ROUND_CEIL)
}
```